okio/jvm/src/main/java/okio/ByteString.kt

```diff
@@ -514,3 +514,38 @@ internal constructor(
     }
   }
 }
+
+/**
+ * Returns a [ByteString] containing a copy of this [ByteArray].
+ *
+ * @see ByteString.of
+ */
+@Suppress("NOTHING_TO_INLINE") // Alias to public API
+inline fun ByteArray.toByteString(offset: Int = 0, byteCount: Int = size): ByteString =
+    ByteString.of(this, offset, byteCount)
+
+/**
+ * Returns a [ByteString] containing a copy of this [ByteBuffer].
+ *
+ * @see ByteString.of
+ */
+@Suppress("NOTHING_TO_INLINE") // Alias to public API
+inline fun ByteBuffer.toByteString(): ByteString = ByteString.of(this)
```

Comment:
Yeah, I like this approach, I'll update the PR

Reply:
Think I misunderstood what you were trying to say, moving these into the companion object won't work since they are extension functions. What was the idea?

Reply:
They still work, you just have to explicitly import them.

The original functions in the companion should be changed to be extensions so that these become redundant and can be deleted.

Reply:
Oh gotcha, thanks